terraform/modules/application/parameters.tf

```diff
@@ -0,0 +1,10 @@
+data "aws_ssm_parameter" "partial_database_connection_string" {

```

Comment:
I was unsure if this was OK / there's a better way of building the complete database connecting string in the application module. 

My concern was whether this might expose the partial database connecting string to the state file (which contains the db endpoint, username and password)? It might be encrypted however so not directly useful? Any info / thoughts on this appreciated 😀

Reply:
I haven't applied this to staging yet until we're sure what it will / won't expose

Reply:
I don't think this changes anything about what ends up in the terraform state - the db password, endpoint etc will already be there (but are not useful unless you can first get access to somewhere the db will accept connections from).

Reply:
Thanks for explaining, I'll leave it as is 😊